app/src/organisms/Desktop/Devices/ProtocolRun/ProtocolRunHeader/DisplayRunStatus.tsx

```diff
@@ -1,46 +1,56 @@
 import { useTranslation } from 'react-i18next'
 
-import { RUN_STATUS_RUNNING } from '@opentrons/api-client'
-import {
-  ALIGN_CENTER,
-  COLORS,
-  Flex,
-  Icon,
-  SPACING,
-  StyledText,
-} from '@opentrons/components'
+import { RUN_STATUS_AWAITING_RECOVERY } from '@opentrons/api-client'
+import { ALIGN_CENTER, Chip, Flex } from '@opentrons/components'
 
 import type { RunStatus } from '@opentrons/api-client'
+import type { ChipType, IconName } from '@opentrons/components'
 
 interface DisplayRunStatusProps {
   runStatus: RunStatus | null
 }
 
+// Styles the run status chip.
+const getRunStatusChip = (
+  runStatus: RunStatus | null
+): [ChipType, boolean, boolean, IconName?] => {
+  const run_status_string = `${String(runStatus)}`
+  switch (run_status_string) {
+    case 'canceled':
+    case 'paused':
+    case 'idle':
+      return ['neutral', false, false]
+    case 'failed':
+    case 'awaiting-recovery':
+      return ['error', true, false]
+    case 'running':
+      return ['success', true, true, 'small-circle']
```

Comment:
```suggestion
      return ['success', true, true, 'circle']
```
See comments on the `icon-data` file for an alternative way to handle this.